src/snarky.d.ts

```diff
@@ -170,22 +265,22 @@ declare class Field {
    * let sum = a.add(5)
    * ```
    */
-  add(y: Field | number | string | boolean): Field;
+  add(y: Field | number | string | bigint): Field;
```

Comment:
I think nobody ever used a `boolean` to specify a field element, anywhere. I removed this feature and made sure you can use `bigint` everywhere instead